rosetta/validator/account.go

```diff
@@ -29,10 +29,12 @@ func (v *Validator) Account(account identifier.Account) error {
 	// possible that the characters are not valid hex encoding.
 	bytes, err := hex.DecodeString(account.Address)
 	if err != nil {
-		return failure.InvalidAccount{
-			Address: account.Address,
-			Chain:   v.params.ChainID.String(),
-			Message: "account address is not a valid hex-encoded string",
+		return errors.InvalidAccount{
+			Description: "account address is not a valid hex-encoded string",
+			Details: []errors.Detail{
+				errors.WithString("address", account.Address),
+				errors.WithString("chain", v.params.ChainID.String()),
+			},
```

Comment:
At this point, I think it would be better to create a construction function and move this explicit code to the failure package.